test/m1_facilitator/models/ERC20GatewayTokenPair/util.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import ERC20GatewayTokenPair from '../../../../src/m1_facilitator/models/ERC20GatewayTokenPair';
+import assert from '../../../test_utils/assert';
+
+export interface ERC20GatewayTokenPairAttributes {
```

Comment:
This file implements the assert function in a different way as compared to what we had been using.
Please let me know if there is an issue in the existing implementation? 
We should keep it consistent.

Reply:
The current implementation allows fine grained assertion possibility. One could specify only the attributes that should be asserted.

Reply:
ok